tests/console/lvm_thin_check.pm

```diff
@@ -1,4 +1,4 @@
-# SUSE's openQA tests
+# SUSE'S openQA tests
```

Comment:
Hate to bother, but why do we want this change? I don't see a reason to make `S` uppercase here and it will be inconsistent then in comparison to other files.